core/shared/protobuf_configmap_reconciler.go

```diff
@@ -72,10 +72,13 @@ func (p *protoBufConfigMapReconciler) createRequiredResources(runtimeInstance ap
 	if err != nil {
 		return nil, err
 	}
+	if protoBufConfigMap!=nil{
```

Comment:
Hi, can you please fix the indentation?

Reply:
Hello,I have made the suggested changes.